src/pages/admin/reports/components/program-expenses-section/components/program-expenses-toolbar/ProgramExpensesToolbar.module.scss

```
@@ -0,0 +1,142 @@
+
+.exchange-rate-label {
+    @include type.small-text-medium;
+    display: flex;
+    align-items: center;
+    letter-spacing: 0.1px;
+    color: c.$grey-900;
+}
+
+.exchange-rate-value {
+    @include type.small-text-medium;
+    box-sizing: border-box;
```

Stylelint (17.6.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)